app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -7,42 +7,138 @@ def params_for_create
+                      {
+                        :component  => "select-field",
+                        :name       => "endpoints.default.default_security_protocol",
+                        :label      => _("Security Protocol"),
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                        :options    => [
+                          {
+                            :label => _("SSL without validation"),
+                            :value => "ssl-no-validation"
+                          },
+                          {
+                            :label => _("SSL"),
+                            :value => "ssl-with-validation"
+                          },
+                          {
+                            :label => _("Non-SSL"),
+                            :value => "non-ssl"
+                          }
+                        ]
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "endpoints.default.default_hostname",
+                        :label      => _("Hostname (or IPv4 or IPv6 address)"),
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                      },
+                      {
+                        :component    => "text-field",
+                        :name         => "endpoints.default.default_api_port",
+                        :label        => _("API Port"),
+                        :type         => "number",
+                        :initialValue => 443,
+                        :isRequired   => true,
+                        :validate     => [{:type => "required-validator"}],
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "endpoints.default.default_userid",
+                        :label      => "Username",
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                      },
+                      {
+                        :component  => "text-field",
+                        :name       => "endpoints.default.password",
+                        :label      => "Password",
+                        :type       => "password",
+                        :isRequired => true,
+                        :validate   => [{:type => "required-validator"}],
+                      }
+                    ]
+                  }
+                ]
+              },
+              {
+                :component => 'tab-item',
+                :name      => 'events',
+                :title     => _('Events'),
+                :fields    => [
+                  {
+                    :component    => 'select-field',
+                    :name         => 'event_stream_selection',
```

Comment:
This should be adjusted after we have the schema changes, this `event_stream_selection` doesn't exist.